README.md

```diff
@@ -2,6 +2,72 @@
 
 `image-tools` is a collection of tools for working with the [OCI image format specification](https://github.com/opencontainers/image-spec).
 
+# Install
+
+Recommend to use `go` to download.
+
+```
+$ go get github.com/opencontainers/image-tools/
+$ cd $GOPATH/src/github.com/opencontainers/image-tools/
+$ make all
+$ sudo make install
+```
+
+## uninstall
+
+```
+$ sudo make uninstall
+```
+
+## example
+
+Firstly, May be you need to install `skopeo`  and more information at [here](https://github.com/projectatomic/skopeo).
```

Comment:
With the new “Obtaining an image” linking to skopeo's installation section, I don't think we need any of the text between the “example” and “Obtaining an image” headers anymore.  Folks who don't have `skopeo` can go over to that project's installation docs and see how to install it without us duplicating their instructions here.